libs/components/theme/src/lib/theming/theme.service.ts

```diff
@@ -14,22 +14,22 @@ export class SkyThemeService {
    * Notifies consumers when the current theme settings have changed.
    */
   public get settingsChange(): Observable<SkyThemeSettingsChange> {
-    return this.settingsObs;
+    return this.#settingsObs;
```

Comment:
I know this isn't your fault, but since this is part of a getter, this should be renamed to `#_settingsObs`.

Reply:
Fixed